toolkit/src/test/kotlin/com/izivia/ocpp/integration/test/IntegrationTestCSApi.kt

```diff
@@ -136,6 +136,7 @@ import com.izivia.ocpp.operation.information.OperationExecution
 import com.izivia.ocpp.operation.information.RequestMetadata
 import com.izivia.ocpp.operation.information.RequestStatus
 import com.izivia.ocpp.OcppVersion
```

Comment:
We need this enum because we use it to initialize a websocket server directly with the ocpp-wamp module
```kotlin
@BeforeEach
    fun init() {
        server = OcppWampServer.newServer(port, setOf(OcppVersion.OCPP_1_6, OcppVersion.OCPP_2_0))
```